installer/x86_64/install.sh

```diff
@@ -20,6 +20,21 @@ _trap_push() {
 }
 _trap_push true
 
+read_conf_file() {
+    local conf_file=$1
+    while IFS='=' read -r var value
+    do
+        f_char=$(echo $var | cut -c1)
```

Comment:
Do you assume the `#` appear as first char of a line? If yes, can you assume it could appear anywhere on a line?

Reply:
Your suggestion is helpful. I modify this part to remove the string behind `#`.